src/@context/Profile.tsx

```diff
@@ -186,7 +186,7 @@ function ProfileProvider({
           cancelTokenSource.token
         )
         setAssets(result.results)
-        setAssetsTotal(result.totalResults)
+        setAssetsTotal(result.results.length)
         LoggerInstance.log(
           `[profile] Fetched ${result.totalResults} assets.`,
```

Comment:
this needs to be changed too, based on above change, `result.totalResults` won't work here